examples/other_resources/03_must-gather-operator.ClusterRoleBinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Curious why this file and 06...CRB were duplicated in examples. Is that so a test deploy only needs to apply files from one directory?